sdk/eventhub/azure-eventhub/stress/templates/testjob.yml

```diff
@@ -0,0 +1,16 @@
+{{- include "stress-test-addons.deploy-job-template.from-pod" (list . "stress.python-eh-stress") -}}
+{{- define "stress.python-eh-stress" -}}
+metadata:
+  labels:
+    testName: "deploy-python-eh-stress"
+    testInstance: "eventhub-{{ .Release.Name }}-{{ .Release.Revision }}"
```

Comment:
where are `.Release.Name` and `.Release.Revision` coming from? helm chart?

Reply:
Correct, the script runs a command like `helm install <release name> <chart path>`. The revision gets incremented if there is already a release with that name deployed into the cluster. This scheme ensures that there is always unique naming for resources and labels, to ensure things don't overlap and chaos isn't mis-targeted.

Reply:
`version`, `appVersion` and `apiVersion` are helm specific fields that we don't really need but have to supply. `name` is also helm-specific, but in the `deploy-stress-tests.ps1` script we use that value to determine the name of the helm release.

So `Chart.yaml -> name` == `Release.Name` and if you run the script multiple times, then `Release.Revision` will increment.